### PR TITLE
[Feature] Add Vibe converter

### PR DESCRIPTION
**Pull request by pangyyyyy**

### CODE REVIEW

File: mmhuman3d/data/data_converters/vibe.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        num_data = len(image_path_)
+
+        if mode == 'mpi_inf_3dhp':
+            image_path_ = np.array([
+                s.replace('imageFrames/', '').replace('frame_', '')
+                for s in data['imgname']
+            ])
+
+        # center scale to bbox
+        w = h = data['scale'] * 200
```
Comment (caizhongang) [inline mmhuman3d/data/data_converters/vibe.py:118]: Just to confirm this `200` is to off-set the magic number in the original dataset?

**Review by caizhongang — COMMENTED** [review]
Currently, we need specific converters for specific algorithms. Will we refactor the converters (in the future) to make them more general and shared amongst algorithms?

File: mmhuman3d/data/data_converters/vibe.py (hunk 1 above)
Comment (pangyyyyy) [reply mmhuman3d/data/data_converters/vibe.py:118]: Yes, this is to account for data with pixel_std=200 in SPIN preprocessed datasets

File: docs/contribute_dataset.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md#lsp
+
+
+### 5. Check that the converter works
+
+
+Check that running this command
+
+```bash
+python tools/convert_datasets.py \
```
[the remark this answers is not in the document]
Comment (pangyyyyy) [reply docs/contribute_dataset.md:130]: Yups, I think step 3 instructs the user to add to `DATASET_CONFIGS`. 
https://github.com/pangyyyyy/mmhuman3d/blob/vibe_converter/docs/contribute_dataset.md#3-add-your-dataset-configuration-to-dataset-configs-under-mmhuman3dtoolsconvert_datasetspy

File: docs/contribute_dataset.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    """
+    ACCEPTED_MODES = ['test', 'train']
+
+    def __init__(self, modes: List = []) -> None:
+        super(LspConverter, self).__init__(modes)
+
+    def convert_by_mode(self, dataset_path: str, out_path: str,
+                        mode: str) -> dict:
+
+```
```
[the remark this answers is not in the document]
Comment (pangyyyyy) [reply docs/contribute_dataset.md:52]: added in 4c9bedf

File: docs/contribute_dataset.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
+## How to add a dataset converter to mmhuman3d
+
```
[the remark this answers is not in the document]
Comment (pangyyyyy) [reply docs/contribute_dataset.md:2]: added in 4c9bedf

File: docs/contribute_dataset.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pangyyyyy) [reply docs/contribute_dataset.md:133]: added a commend for dataset-name in 4c9bedf , hope this helps to clarify

**Review by caizhongang — APPROVED** [review]

**Pull request merged**